templates/activity_stream/impression_stats_flat/view.sql

```diff
@@ -0,0 +1,13 @@
+CREATE OR REPLACE VIEW
+  `moz-fx-data-shared-prod.activity_stream.impression_stats_flat`
+AS
+SELECT
+  tile_id,
+  tile_id_types.type AS tile_type,
+  stats.* EXCEPT(tile_id)
+FROM
+  `moz-fx-data-shared-prod.activity_stream_derived.impression_stats_flat_v1` AS stats
+LEFT JOIN
+  `moz-fx-data-shared-prod.activity_stream.tile_id_types` AS tile_id_types
```

Comment:
My understanding is that the `tile_id_types` view exists in order to enrich the impression_stats data and we can do the join as part of this view with negligible impact since `tile_id_types` is so small.

@kirill-demtchouk, does this change look good to you? It makes a `tile_type` column available when querying `activity_stream.impression_stats_flat`.

Reply:
@jklukas Looks good to me. Good call!